docs/research/power-consumption-tests.md

```diff
@@ -0,0 +1,160 @@
+
+Latency (ms):
+         min:                                    6.40
+         avg:                                    7.41
+         max:                                    8.13
+         95th percentile:                        7.70
+         sum:                                74064.78
+
+Threads fairness:
+    events (avg/stddev):           2500.0000/18.61
+    execution time (avg/stddev):   18.5162/0.00
+```
+
+![sysbench-voltage](../images/cb4-sysbench-voltage.png)
+
+![sysbench-current](../images/cb4-sysbench-current.png)
+
+![sysbench-power](../images/cb4-sysbench-power.png)
+
+##### Conclusions
+
+In idle state, when only Debian is running without any commands and
+applications, power consumption oscillates around 4W. It increases to almost 6W
+(~5.8W) when `sysbench` is running. That change is mainly caused by current
+rise. However, simultaneously there is a little voltage drop on the bus. As you
+can see it drops from about 12.18 V to 12.12 V. That behavior is normal - the
+greater the load, the greater voltage decrease can be observed. It is rather
+related to quality of power supply and wires than to end device. Moreover, as
+this voltage drop is slight, it can be omitted in analysis.
+
+#### Results with coreboot v4.11.0.5
+
+Above test was repeated with `coreboot v4.11.0.5` which has **ASPM** (Active
+State Power Management) enabled. Exactly same test case was carried out and same
+measurements were taken.
+
+```bash
+$ sysbench cpu run --threads=4 --events=10000 --cpu-max-prime=20000 --time=0 --validate
+sysbench 1.0.19 (using bundled LuaJIT 2.1.0-beta2)
+
+Running the test with following options:
+Number of threads: 4
+Validation checks: on.
+
+Initializing random number generator from current time
+
+
+Prime numbers limit: 20000
+
+Initializing worker threads...
+
+Threads started!
+
+CPU speed:
+    events per second:   539.80
+
+General statistics:
+    total time:                          18.5200s
+    total number of events:              10000
+
+Latency (ms):
+         min:                                    6.36
+         avg:                                    7.41
+         max:                                    8.17
+         95th percentile:                        7.70
+         sum:                                74053.97
+
+Threads fairness:
+    events (avg/stddev):           2500.0000/18.79
+    execution time (avg/stddev):   18.5135/0.00
+```
+
+![sysbench-voltage-2](../images/cb5-sysbench-voltage.png)
+
+![sysbench-current-2](../images/cb5-sysbench-current.png)
+
+![sysbench-power-2](../images/cb5-sysbench-power.png)
+
+##### coreboot v4.11.0.4 vs v4.11.0.5
+
+Comparing those results to previous one (with `coreboot v4.11.0.4`), following
+differences can be noticed:
+
+- average bus voltage level in idle state and during stress-test is slightly
+  larger with `coreboot v4.11.0.5` than `coreboot v4.11.0.4`; as those
+  differences are very small, it can be treated as irrelevant;
```

Comment:
@piotr-kleins how small in %? I would avoid using work "irrelevant" since this is not an engineering term, maybe those are below error margin or sth more mathematical.

Reply:
Replace it with percentage expression.